src/main/scala/com/atomist/rug/runtime/js/RugContext.scala

```diff
@@ -44,4 +58,6 @@ object LocalRugContext extends RugContext {
     * Id of the team we're working on behalf of
     */
   override def teamId: String = "LOCAL_CONTEXT"
+
+  override def contextRoot(): GraphNode = ???
```

Comment:
Could we move LocalContext to test directory? I'm not sure it's used in cli/runner...